pubsub/topics/main.go

```diff
@@ -125,6 +147,34 @@ func publish(client *pubsub.Client, topic, msg string) error {
 	return nil
 }
 
+func publishWithSettings(client *pubsub.Client, topic string, msg []byte) error {
+	ctx := context.Background()
+	// [START publish_settings]
+	t := client.Topic(topic)
+	t.PublishSettings = pubsub.PublishSettings{
+		ByteThreshold:  5000,
+		CountThreshold: 10,
+		DelayThreshold: 100 * time.Millisecond,
+	}
+	result := t.Publish(ctx, &pubsub.Message{Data: msg})
```

Comment:
I avoided to repeat the `result` handling section previously because the previous Receive snippet documented it, but gonna add that section for both of the new samples to make it easier to copy/paste.

re: num of goroutines. It is too much detail in this context. That's why we will only mention it after the document opens a section about concurrency related settings.